src/main/java/com/revature/services/QuestionLogic.java

```diff
@@ -1,5 +1,7 @@
 package com.revature.services;
 
+import com.revature.domain.Eval;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.revature.domain.Eval'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AUselessImportCheck)